__tests__/components/EtherspotApprovalTransaction.test.js

```diff
@@ -19,7 +19,7 @@ describe('EtherspotApprovalTransaction', () => {
 
   it('throws error if wrong receiver address provided', () => {
     expect(() => render(
-      <EtherspotUi provider={null}>
+      <EtherspotTransactionKit provider={null}>
```

Comment:
Oops sorry mate, we agreed we was going to rename this to `<EtherspotWrapper />`

https://linear.app/pillarproject/issue/PRO-1235#comment-428ceb53

Reply:
Yeah sorry, I did this because thought maybe `<EtherspotTransactionKit />` would make more sense because we have `<Etherspot />` general provider coming from `@etherspot/react` lib and imagine having `<Etherspot />` and `<EtherspotWrapper />` within same codebase, doesn't read well, right? And `<EtherspotTransactionKit />` matches perfectly the lib name `@etherspot/transaction-kit` while `<EtherspotWrapper />` doesn't speak as much, please review this again and lmk wdyt, thank you!